funtofem/interface/test_solver.py

```diff
@@ -948,6 +948,134 @@ def relative_error(cls, truth, pred):
         else:
             return (pred - truth) / truth
 
+    @classmethod
+    def design_sweep(
+        cls,
+        model,
+        driver,
+        nsweep=10,
+        eps=1e-4,
+        base_folder=None,
+        csv_file_prefix="design-sweep",
+        include_derivatives=True,
+    ):
+        """
+        perform a design sweep on a model and driver to determine the range of function values
+        """
+        import pandas as pd, os
```

Comment:
Can we avoid adding extra dependencies? I think pandas was what caused my anaconda environment issue this past week.

Reply:
It's an optional dependency since it is only imported inside that method. If you don't run that method it isn't imported. That's why I did that

Reply:
Right, but you still need it to use that routine. Is pandas' data frame implementation the only reasonable thing that will work in this case?

Reply:
Alright, I can manually write the csv file

Reply:
I'm not super opposed to using pandas here in this limited capacity. But if it's not too much trouble to do it without pandas, then it would be better not to have the dependency. On the other hand, if writing your own implementation is going to be too time-consuming or bug-prone, then it may be better to just use pandas. Up to you.

Reply:
Ok I implemented the design sweep method without pandas